code/lib/cli/src/generators/SERVER/index.ts

```diff
@@ -10,7 +10,9 @@ const generator: Generator = async (packageManager, npmOptions, options) => {
   });
 
   const templateDir = join(getCliDir(), 'templates', 'server');
```

Comment:
We can just get rid of this block of code entirely. I just tested it out on a sample project and will be adding a telescoping PR to do it.